tests-legacy/Unit/Adapter/Module/Configuration/ModuleSelfConfiguratorTest.php

```diff
@@ -214,10 +214,10 @@ public function testSqlStep()
 
         $this->assertTrue($this->getModuleSelfConfigurator()->module($name)->file($filepath)->configure());
         // Check files are equals
-        $this->assertTrue(in_array('TRUNCATE TABLE `ps_doge_army`', $this->connection->executedSql));
-        $this->assertTrue(in_array('UPDATE `ps_doge` SET `wow` = 1', $this->connection->executedSql));
-        $this->assertFalse(in_array('UPDATE `ps_doge` SET `wow` = 1;', $this->connection->executedSql));
-        $this->assertTrue(in_array('TRUNCATE TABLE `ps_lolcat_army`', $this->connection->executedSql));
```

Comment:
This obviously breaks the tests. We're verifying here what SQL requests ran on the database, so the prefix must be applied.

Reply:
my bad, I close this PR :)